lib/numbers.md

```diff
@@ -5,20 +5,27 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What are the different kinds of numbers in Ruby?
 
-Replace me with your response...
+Fixnum are integers and Float are decimals
 
 # What are some common operations and comparisons you would perform on numbers?
 
-Replace me with your response...
+Addition, subtraction, multiplication, division, to the power of, modulo, and absolute value are common operations.  
+is less than, is greater than, is less than or eaqual to, is greater than or equal to, is equal to, is not equal to are common comparions.
 
 # What is the difference between the `+` operation on a number versus on a String?
 
-Replace me with your response...
+The '+' used with numbers is the addition operation.  2 + 2 => 4
+The '+' used with strings connects the two together. "2" + "2" => 22
 
 # If you have a _String_ `"20"` and want to perform a mathematical operation (like division or multiplication) on it, will it work? If yes, why? If not, how would you make it work?
 
-Replace me with your response...
+It will not work until you convert the string to either an integer to_i or to a float to_f.
```

Comment:
What result to you get when you run "20" * 3? Does this operation still work, even if it doesn't work the way we'd expect/hope it to?]